tarantool_test.go

```diff
@@ -352,6 +354,31 @@ func BenchmarkClientParallelMassiveUntyped(b *testing.B) {
 	close(limit)
 }
 
+func BenchmarkSQLParallel(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+
+	_, err = conn.Replace(spaceNo, []interface{}{uint(1111), "hello", "world"})
+	if err != nil {
+		b.Errorf("No connection available")
+	}
```

Comment:
Is it not the same space as used in the `SELECT` query below (`SQL_TEST`)?

Reply:
Yes, the `spaceNo` was from another one, fixed. Thank you.